source/patterns/04-pages/research-article/research-article~ja.yaml

```diff
@@ -1,5 +1,11 @@
 lang: ja
 
+siteHeader:
+  logo:
+    href: '#'
+    image:
+      alt: 'Liberoのロゴ'
```

Comment:
As before are these rogue characters in the alt?

Reply:
See https://github.com/libero/pattern-library/pull/97#discussion_r285908339